qualtran/bloqs/basic_gates/rotation.py

```diff
@@ -30,16 +31,18 @@
     from qualtran.resource_counting import BloqCountT, SympySymbolAllocator
 
 
-@frozen
-class RotationBloq(Bloq, metaclass=abc.ABCMeta):
-    angle: float
-    eps: float = 1e-11
+class _HasEps(Protocol):
+    """Protocol for typing `RotationBloq` base class mixin that has accuracy specified as eps."""
 
+    eps: float
+
+
+class RotationBloq(GateWithRegisters, metaclass=abc.ABCMeta):
```

Comment:
should this be private, i.e. `_RotationBloq`? Or should this be non-abstract to count arbitrary rotations?

Reply:
I think it should be private for now; I'll make it private

Reply:
done